src/meteors/visualize/attr_visualize.py

```diff
@@ -71,6 +72,8 @@ def visualize_attributes(
     ax[0, 0].grid(False)
     ax[0, 0].axis("off")
 
+    if title is None:
+        title = f"HSI Attributes of: {rotated_attributes_dataclass.attributes}"
```

Comment:
title is not assigned to anything. Additionally you need to consider if the user passed `ax` or None because then we can set title on fig or only on ax